test/spec/invertedIndexSpec.js

```diff
@@ -1,27 +1,21 @@
-const validFile = require('./books/validFile.json');
-const simpleValidFile = require('./books/simpleValidFile.json');
+const books = require('./books/validFile.json');
+const shortStories = require('./books/simpleValidFile.json');
 const empty = require('./books/empty.json');
 const emptyText = require('./books/emptyText.json');
 const notArray = require('./books/notArray.json');
 const wrongData = require('./books/wrongData.json');
 
 const index = new InvertedIndex();
+const filename = 'shortStories.json';
 
 describe('InvertedIndex', () => {
-  beforeAll(() => {
```

Comment:
"Ubiquitous Language," a body of knowledge that can be understood by both the developer and the customer. This ubiquitous language is used to shape and develop the requirements and testing needed, at the level of the customer's understanding.

```
describe("Calculator: add", () => {

    it("should be able to add 2 numbers together", () => {
        var total = add(1, 2);
        expect(total).toBe(3);
    });

    it("should be able to add 3 numbers together", () => {
        var total = add(1, 2, 3);
        expect(total).toBe(6);
    });
});
```
Please review your test paying more attention to the language used on your "it" and 'describe' functions